Janelas/Popups/ExhangeRates.xaml.cs

```diff
@@ -22,7 +22,7 @@ public ExchangeRates(decimal exchangeRatePontosToMiningCoin)
                     {
                         this.Close();
 
-                        MessageBox.Show("Wait for Dashboard load first"); return;
+                        MessageBox.Show("Aguarda carregar, porra"); return;
```

Comment:
Tenho que deixar em inglês enquanto não for feito um sistema multi-idiomas

Reply:
Eu não havia visto a questão do DoNotUpdate, aí nenhuma alteração no código tinha efeito, daí alterei a mensagem como teste kkkkk